pkg/cluster/px/px.go

```diff
@@ -53,7 +53,7 @@ const (
 
 var (
 	pxVersionRegex         = regexp.MustCompile(`^(\d+\.\d+).*`)
-	ociMonImageRegex       = regexp.MustCompile(".*registry.connect.redhat.com/portworx/px-enterprise.+|.+oci-monitor.+")
+	ociMonImageRegex       = regexp.MustCompile(`.*registry.connect.redhat.com/portworx/(px-enterprise|px-monitor).+|.+oci-monitor.+`)
```

Comment:
I'm surprised this one works -- note that it's more common to pack _all_ the alternative matches between the brackets, e.g.:
```.../(px-enterprise|px-monitor|.+oci-monitor).+```

Reply:
The current change is working. Here's the test: https://play.golang.org/p/v72DfdyE1pd